content/docs/dvclive/ml-frameworks/pytorch-lightning.md

```diff
@@ -61,6 +61,23 @@ checkpointing at all as described in the
 
 - `prefix` - (`None` by default) - string that adds to each metric name.
 
+- `log_model` - (`False` by default) - use
+  [`live.log_artifact()`](/doc/dvclive/live/log_artifact) to log checkpoints
+  created by
+  [`ModelCheckpoint`](https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.callbacks.ModelCheckpoint.html)
+  and annotate the best checkpoint with `type=model` and `name=best` for use in
+  [Studio model registry]. DVCLive will <abbr>cache</abbr> the checkpoint
```

Comment:
It took me a while to understand the part about cache and being able to recover it. I think most of the ppl will be confused by this (I expect they don't know much about DVC mechanics). Two suggestions:

- remove it
- try to be more explicit: DVCLive saves checkpoints with DVC - it means they are cached in DVC cache, also could be pushed to remote storage, etc. It also means that DVCLive drops the directory (it's safe) when it runs a new experiments. Model weights for any experiment are stored in DVC and can be restored ...

------------------

Btw, what happens, if `save_dvc_exp` is False? How do I recover those?